### PR TITLE
docs: update landing image for improved clarity

### PR DESCRIPTION
**Pull request by SAF2k**
## Description
This commit resolves the issue of a blurry landing image in our documentation (#3827). I've replaced the image with a higher-resolution version to enhance clarity and improve the overall user experience. The new image ensures that our documentation content is displayed crisply and clearly. Please review the updated image placement to confirm its alignment with our documentation's visual standards.

## Markup:
image

### CODE REVIEW

**Comment by SAF2k** [comment]
This commit addresses the issue reported in VaDropdown example #3826, where the dropdown component was overflowing on mobile screens. To fix this problem, I have applied responsive design adjustments to ensure the dropdown fits correctly within the mobile screen's viewport. Please review the changes and verify that the dropdown now displays correctly on mobile devices without overflowing the screen. Your feedback is valuable to ensure a seamless user experience.

**Comment by SAF2k** [comment]
This commit resolves an overflow issue in the VaPagination example #3824 , ensuring the pagination component remains within the viewport on different screen sizes. Please review.

File: packages/docs/page-config/ui-elements/pagination/examples/LimitVisible.vue [hunk 1]
```diff
@@ -3,6 +3,7 @@
     v-model="value"
     :pages="15"
     :visible-pages="4"
+    class="justify-center sm:justify-start"
```
Comment (m0ksem) [inline packages/docs/page-config/ui-elements/pagination/examples/LimitVisible.vue:6]: It still overflows in some cases, but it looks so much better than before. I think we need to make paddings smaller on mobile screen and add scroll to example card. But it is in a different issue!

**Review by m0ksem — APPROVED** [review]

**Comment by m0ksem** [comment]
Thanks!

**Pull request merged**